airflow/api_connexion/openapi/v1.yaml

```
@@ -1996,6 +2031,14 @@ components:
       required:
         - dag_id
 
+    UpdateDagRunState:
+      type: object
+      properties:
+        state:
+          $ref: '#/components/schemas/DagState'
```

```suggestion
          description: The state to set this DagRun
          type: string
          enum:
              - failed
              - success
```
Instead of using the DagState, we should use enums to avoid setting DagRun state to running or queued which are DagStates